christfellowshipchurch/web-app-v2
# Added iconCollection component

### About
This is a component that reflects a collection of icons(images) that is reusable across our platform.

### Test Instructions
To test this component you can: 
-See it in storybook.
-Through this branch, use the IconCollection component in any of our pages.

### Screenshots

<img width="342" alt="Screen Shot 2021-10-13 at 9 20 06 AM" src="https://user-images.githubusercontent.com/46769629/137330506-e25d9265-7a32-49d0-80ab-4d9bbdafb803.png">

<img width="1432" alt="Screen Shot 2021-10-12 at 5 46 58 PM" src="https://user-images.githubusercontent.com/46769629/137330500-c2914f57-7381-4f36-b8a2-3260df32ced0.png">

### Closes Tickets
[CFDP-1718](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-1718)


### Related Tickets
N/A
